src/context/auth-context.js

```diff
@@ -0,0 +1,25 @@
+import { useState, useContext, createContext, useEffect } from "react";
+
+const AuthContext = createContext();
+
+const AuthProvider = ({ children }) => {
+  const [auth, setAuth] = useState({});
+  useEffect(() => {
+    (async () => {
+      const token = localStorage.getItem("token");
+      token
+        ? setAuth({ token: token, isLoggedIn: true })
+        : setAuth({ token: "", isLoggedIn: false });
+    })();
```

Comment:
i dont think a async function is required here because as per my knowledge local storage calls are synchronous

Reply:
Thanks Omkar, will make the changes.